superset/assets/cypress/integration/explore/visualizations/table.js

```diff
@@ -0,0 +1,92 @@
+import { FORM_DATA_DEFAULTS, NUM_METRIC, SIMPLE_FILTER } from './shared.helper';
+
+// Table
+
+describe('Table chart', () => {
+  const VIZ_DEFAULTS = { ...FORM_DATA_DEFAULTS, viz_type: 'table' };
+
+  beforeEach(() => {
+    cy.login();
+    cy.server();
+    cy.route('POST', '/superset/explore_json/**').as('getJson');
+  });
+
+  it('Test table with adhoc metric', function () {
```

Comment:
you have a mix of `function () {}` and `() => {}`, most of our tests I think use the latter / there's no benefit to the former unless you provide a name to the function for debugging. (`prettier` would also use the latter).

Reply:
Bah sorry I forgot I didn't go back and fix old files, let me do that.